docs/sources/setup-grafana/configure-grafana/enterprise-configuration/index.md

```diff
@@ -464,7 +464,7 @@ The default is `25`.
 
 ### url
 
-The full Redis URL of your Redis server. For example: `redis://username:password@localhost:6739/0`. To enable TLS, use the `redis` scheme.
+The full Redis URL of your Redis server. For example: `redis://username:password@localhost:6379`. To enable TLS, use the `rediss` scheme.
```

Comment:
I know this is a backport, but should the Rediss URL start with `reddis://...`? I think it can be written either way, but I'm not sure.